hypervisor/src/vm.rs

```diff
@@ -231,6 +231,11 @@ pub enum HypervisorVmError {
     ///
     #[error("Failed to complete isolated import: {0}")]
     CompleteIsolatedImport(#[source] anyhow::Error),
+    #[cfg(feature = "mshv")]
+    /// Failed to set partition property
+    ///
+    #[error("Failed to set partition property: {0}")]
+    SetPartitionProperty(#[source] anyhow::Error),
```

Comment:
I suspect we don't really want to define an error type for all the lower-level things that could go wrong here in the generic code.

IMO, only the `mshv` hypervisor code should know about 'partition's, etc. The generic CLH code can just see it as 'failed to pause/unpause', and the wrapped error will contain the additional details from the `mshv` code...

Also worth noting: we already have a `SetPartitionProperty` error in hypervisor/src/hypervisor.rs, which is clearly in the wrong place.

Reply:
I can cleanup in another PR

Reply:
Yes, please don't have hypervisor specific errors - if you can avoid it at the top level.